cmd/jobsink/main.go

```diff
@@ -313,6 +313,10 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				MountPath: "/etc/jobsink-event",
 			})
 		}
+		job.Spec.Template.Spec.Containers[i].Env = append(job.Spec.Template.Spec.Containers[i].Env, corev1.EnvVar{
+			Name:  "K_EVENT_PATH",
+			Value: "/etc/jobsink-event",
+		})
```

Comment:
The value you are using here isn't guaranteed to be correct. If you look at the logic above, a different volume mount may be chosen (which could have a different mount path)

Reply:
@Cali0707 : i have updated the logic,
If the Volume mount already exists, the code is taking the volume mount path  from the job
If the Volume mount is not there, the code is taking the defined mount path